docs/modules/ROOT/nav.adoc

```diff
@@ -28,6 +28,12 @@ include::user:ROOT:partial$shared-nav.adoc[]
 ** xref:exoscale-dbaas/opensearch/delete.adoc[]
 ** xref:exoscale-dbaas/opensearch/usage.adoc[]
 
+.VSHN managed
```

Comment:
Please name this "Managed by VSHN" to adhere to the naming in https://products.docs.vshn.ch/products/appcat/index.html